### PR TITLE
Chris vasquez

### PR DESCRIPTION
**Pull request by Chrvasq**
Added styling to Navigation and Footer sections.
Updated about.html to reflect the styling of Navigation and Footer sections.
Updated displayed text from About Us to About on both pages.
Added media query variables for desktop and tablet breakpoints.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -34,24 +34,23 @@ <h2>Meet the team behind Use My Tech Stuff</h2>
...
-              </section>
-          </nav>
-
-          <section class="copyright">
-              <p>© TSTUFF 2019</p>
-          </section>
-      </footer>
+    <footer>
+            <nav>
+                <a href="#">Home</a>
```
Comment (shockin779) [inline about.html]: This should point to index.html

File: about.html [hunk 2]
```diff
@@ -34,24 +34,23 @@ <h2>Meet the team behind Use My Tech Stuff</h2>
...
+                <a href="about.html">About</a>
```
Comment (shockin779) [inline about.html]: This should point to #

File: about.html (hunk 1 above)
Comment (Chrvasq) [reply about.html]: Updated

File: about.html (hunk 2 above)
Comment (Chrvasq) [reply about.html]: Updated

**Pull request merged**